build.gradle

```diff
@@ -9,13 +9,13 @@ buildscript {
         // Version used for all submodule artifacts.
         // Snapshot publishing changes (or adds) the suffix after '-' with 'SNAPSHOT' prior to publishing.
         // The 'publishSigned' task publishes to SonaType's staging repo and 'publishSnapshot' instantly uploads to the snapshots repo.
-        globalVersion = '1.0.0-alpha.22'
+        globalVersion = '1.0.0-alpha.23'
 
         versions = [
             // Kotlin multiplatform versions.
             kotlin:'1.4.10',
-            serialization:'1.0.0-RC2',
-            coroutines:'1.3.9',
+            serialization:'1.0.0',
+            coroutines:'1.4.0-M1',
```

Comment:
Why did you update `coroutines` to `1.4.0-M1`? Is there something wrong with version `1.3.9`?

Reply:
It is the version they claim is aligned with Kotlin 1.4.0. Since we are on Kotlin 1.4.0, I believe it makes sense to do so.

In general, I try to upgrade dependencies each release. That way we catch changes sooner and upgrading becomes a smaller task. For example, upgrading dokka has been postponed for ages since it is a big task. :(

Reply:
Version `1.3.9` is already aligned with kotlin 1.4.0 see [link](https://github.com/Kotlin/kotlinx.coroutines/releases/tag/1.3.9). `1.4.0-M1` isn't a production release and it does contain some breaking changes. Therefor I think it makes sense to keep `1.3.9` until `1.4.0` is released.

Reply:
Okay. Let me know when you need this update. Will leave it at 1.3.9 until I hear a request.